pkg/window/strategy/sliding/sliding.go

```diff
@@ -183,6 +192,19 @@ func (w *Windower) CloseWindows(time time.Time) []*window.TimedWindowRequest {
 		windowOperations = append(windowOperations, operation)
 		w.closedWindows.InsertBack(win)
 	}
+
+	metrics.ActiveWindowsCount.With(map[string]string{
```

Comment:
Just out of curiosity, why do we choose to emit metrics when we call CloseWindows, as opposed to other methods that are also updating the active&closed windows?

Reply:
CloseWindows() is invoked once per batch, whereas other functions are invoked for every message. I think it's good to track the windows count once per batch.